docs/source/usage.rst

```diff
@@ -195,48 +205,53 @@ Possible links are::
   :py:attr:`twisted:twisted.protocols.amp.BinaryBoxProtocol.boxReceiver`
 
 
-Building pydoctor together with Sphinx HTML build
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building pydoctor together with Sphinx HTML build or on Read The Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When running pydoctor with HTML generation it will generate a set of static
 HTML files that can be used any HTTP server.
 
 Under some circumstances (ex Read The Docs) you might want to trigger the
 pydoctor API docs build together with the Sphinx build.
 
-This can be done by using the `pydoctor.sphinx_ext.build_apidocs` extension.
+This can be done by using the :py:mod:`pydoctor.sphinx_ext.build_apidocs` extension.
 
 Inside your Sphinx `conf.py` file enable and configure the extension in this
 way.::
 
     extensions.append("pydoctor.sphinx_ext.build_apidocs)
 
     pydoctor_args = [
-        '--quiet',
         '--project-name=YOUR-PROJECT-NAME',
         '--project-url=YOUR-PROJECT-HOME-URL',
         '--docformat=epytext',
         '--intersphinx=https://docs.python.org/3/objects.inv',
-        '--make-html',
         '--html-viewsource-base=https://github.com/ORG/REPO/tree/default',
         '--html-output={outdir}/api',
-        '--project-base-dir=/absolute/path/to/your/project',
-        '/absolute/path/to/your/project/package1'
+        '--project-base-dir=path/to/source/code',
+        'path/to/source/code/package1'
         ]
 
-The `{outdir}` will replaced with the Sphinx build dir.
+    pydoctor_url_fragment = '/en/{rtd_version}/api/'
+
+You can pass almost any argument to `pydoctor_args`
+in the same way you call `pydoctor` from the command line.
 
-You can pass any argument, in the same way you call `pydoctor` from the
-command line.
+You should not pass the `--make-html`, `--make-intersphinx` or `--quiet`
+arguments as this will break the build.
+The extension will add them automatically.
```

Comment:
For my understanding: `--make-html` and `--quiet` are auto-added, while `--make-intersphinx` is not auto-added, but having that option enabled would break the output because no HTML would be produced by pydoctor, correct?

Reply:
Well. Adding those arguments will not break the build .. at least for now.

So if you add them, nothing would happen.

But to leave some space for future changes, I prefer to document it in this way.

Reply:
I feel that this is a dev tools and there is not much value in making the tool bullet-proof.
The whole extension is a hack... quite fragile..but for now it works.

Reply:
My issue is not with the code being fragile, but with the documentation suggesting that the code is more fragile than it actually is. I'd like to think that the extension will become less fragile over time. So if it can handle these arguments now, why would it be a problem to keep it that way in the future?

And if there really is no way to stay compatible, we're likely making such invasive changes that argument lists need to be revised anyway. Do you have a particular scenario in mind where this warning will pay off? Because I can't think of one.